proto/vote/exported/v1beta1/types.proto

```diff
@@ -44,6 +47,8 @@ message PollMetadata {
     (gogoproto.nullable) = false
   ];
   string reward_pool_name = 10;
+  int64 grace_period = 11;
+  int64 completed_at = 12;
```

Comment:
All these numbers eventually do math with block height from cosmos sdk which is int64. To avoid overflow, it would only make sense for them to be uint32 instead of uint64, but then we would have to cast everywhere. Making them int64 is simple and clean. We just need to validate.